docs/DeveloperGuide.md

```diff
@@ -1186,20 +1255,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. Nurse's input is invalid command format.
 
-    * 3a1. PatientSync shows an error message.
+  * 3a1. PatientSync shows an error message.
 
-      Use case resumes at step 2.
+    Use case resumes at step 2.
+
+* 3b. Nurse's input contains invalid patient index or event index.
+
+  * 3b1. PatientSync shows an error message.
 
-**Use case: Delete tag from a patient**
+    Use case resumes at step 2.
+
+**Use case: UC08 - Add tag to a patient**
 
 **MSS**
 
-1.  Nurse requests to list patients
-2.  PatientSync shows a list of patients
-3.  Nurse requests to delete a tag from a specific patient in the list
-4.  PatientSync deletes a tag from the patient
+1. Nurse requests to list patients.
+1. PatientSync shows a list of patients.
+1. Nurse requests to add a tag to a specific patient in the list.
+1. PatientSync adds a tag to the patient and displays success message.
```

Comment:
adds the specified tag(s) and display status message

Reply:
Updated!